engine/ddf/src/ddf/ddf_math.proto

```diff
@@ -5,11 +5,11 @@ import "ddf/ddf_extensions.proto";
 option java_package = "com.dynamo.proto";
 option java_outer_classname = "DdfMath";
 
-option (ddf_includes) = "dmsdk/vectormath/cpp/vectormath_aos.h";
+option (ddf_includes) = "dmsdk/dlib/vmath.h";
```

Comment:
The goal is to at least open up for the possibility of using another vector library.
The current one doesn't support iOS/Android with NEON simd.

Also, having a shorter namespace is good imho.